mappings/net/minecraft/client/gui/Element.mapping

```diff
@@ -1,38 +1,142 @@
+		COMMENT has been captured.
+		COMMENT
+		COMMENT @return {@code true} to indicate that the event handling is successful/valid
+		COMMENT @see net.minecraft.client.Mouse#onMouseScroll(long, double, double)
 		ARG 1 mouseX
+			COMMENT the X coordinate of the mouse
 		ARG 3 mouseY
+			COMMENT the Y coordinate of the mouse
 		ARG 5 amount
+			COMMENT value is {@code > 1} if scrolled down, {@code < 1} if scrolled up
 	METHOD method_25402 mouseClicked (DDI)Z
+		COMMENT Callback for when a mouse button down event
+		COMMENT has been captured.
+		COMMENT
+		COMMENT The button number is identified by the constants in
+		COMMENT {@link org.lwjgl.glfw.GLFW GLFW} class.
+		COMMENT
+		COMMENT @return {@code true} to indicate that the event handling is successful/valid
+		COMMENT @see net.minecraft.client.Mouse#onMouseButton(long, int, int, int)
+		COMMENT @see org.lwjgl.glfw.GLFW#GLFW_MOUSE_BUTTON_1
 		ARG 1 mouseX
+			COMMENT the X coordinate of the mouse
 		ARG 3 mouseY
+			COMMENT the Y coordinate of the mouse
 		ARG 5 button
+			COMMENT the mouse button number
 	METHOD method_25403 mouseDragged (DDIDD)Z
+		COMMENT Callback for when a mouse button drag event
+		COMMENT has been captured.
+		COMMENT
+		COMMENT The button number is identified by the constants in
+		COMMENT {@link org.lwjgl.glfw.GLFW GLFW} class.
+		COMMENT
+		COMMENT @return {@code true} to indicate that the event handling is successful/valid
+		COMMENT @see net.minecraft.client.Mouse#onCursorPos(long, double, double)
+		COMMENT @see org.lwjgl.glfw.GLFW#GLFW_MOUSE_BUTTON_1
 		ARG 1 mouseX
+			COMMENT the current X coordinate of the mouse
 		ARG 3 mouseY
+			COMMENT the current Y coordinate of the mouse
 		ARG 5 button
+			COMMENT the mouse button number
 		ARG 6 deltaX
+			COMMENT the difference of the current X with the previous X coordinate
 		ARG 8 deltaY
+			COMMENT the difference of the current Y with the previous Y coordinate
 	METHOD method_25404 keyPressed (III)Z
+		COMMENT Callback for when a key down event has been captured.
+		COMMENT
+		COMMENT The key code is identified by the constants in
+		COMMENT {@link org.lwjgl.glfw.GLFW GLFW} class.
+		COMMENT
+		COMMENT @return {@code true} to indicate that the event handling is successful/valid
+		COMMENT @see net.minecraft.client.Keyboard#onKey(long, int, int, int, int)
+		COMMENT @see org.lwjgl.glfw.GLFW#GLFW_KEY_Q
+		COMMENT @see org.lwjgl.glfw.GLFWKeyCallbackI#invoke(long, int, int, int, int)
 		ARG 1 keyCode
+			COMMENT the named key code of the event as described in the {@link org.lwjgl.glfw.GLFW GLFW} class
 		ARG 2 scanCode
+			COMMENT the unique/platform-specific scan code of the keyboard input
 		ARG 3 modifiers
+			COMMENT a GLFW bitfield describing the modifier keys that are held down; See {@linkplain https://www.glfw.org/docs/3.3/group__mods.html GLFW Modifier key flags}
 	METHOD method_25405 isMouseOver (DD)Z
+		COMMENT Checks of the mouse position is within the bound
```

Comment:
```suggestion
		COMMENT Checks if the mouse position is within the bound
```

Reply:
Thanks for spotting the error. Will make sure a more careful check next time.